de.bforartists.Bforartists.json

```diff
@@ -0,0 +1,176 @@
+                "--disable-devices",
+                "--enable-muxer=avi",
+                "--enable-muxer=h264",
+                "--enable-muxer=mov",
+                "--enable-muxer=mp4",
+                "--enable-muxer=ogg",
+                "--enable-muxer=webm",
+                "--enable-demuxer=avi",
+                "--enable-demuxer=h264",
+                "--enable-demuxer=mov",
+                "--enable-demuxer=mp3",
+                "--enable-demuxer=ogg",
+                "--enable-demuxer=wav",
+                "--enable-parser=h264",
+                "--enable-parser=vorbis",
+                "--enable-encoder=aac",
+                "--enable-encoder=libtheora",
+                "--enable-encoder=libvorbis",
+                "--enable-encoder=libvpx_vp8",
+                "--enable-encoder=libvpx_vp9",
+                "--enable-encoder=libx264",
+                "--enable-encoder=mpeg4",
+                "--enable-decoder=aac",
+                "--enable-decoder=h264",
+                "--enable-decoder=libvorbis",
+                "--enable-decoder=libvpx_vp8",
+                "--enable-decoder=libvpx_vp9",
+                "--enable-decoder=mp3",
+                "--enable-decoder=mpeg4",
+                "--enable-decoder=pcm_s16le",
+                "--enable-decoder=theora"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.ffmpeg.org/releases/ffmpeg-5.0.1.tar.xz",
+                    "sha256": "ef2efae259ce80a240de48ec85ecb062cecca26e4352ffb3fda562c21a93007b"
+                }
+            ],
+            "cleanup": [
+                "/lib/codecs/include",
+                "/lib/codecs/lib/pkgconfig",
+                "/lib/codecs/share"
+            ]
+        },
+        {
+            "name": "bforartists",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 bforartists.sh /app/bin/bforartists",
+                "mv bforartists /app/bforartists",
+                "install -Dm644 /app/bforartists/bforartists.desktop /app/share/applications/$FLATPAK_ID.desktop",
+                "desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop",
+                "install -Dm644 /app/bforartists/bforartists.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg",
+                "install -Dm644 /app/bforartists/bforartists-symbolic.svg /app/share/icons/hicolor/symbolic/apps/$FLATPAK_ID-symbolic.svg",
+                "cd /app/bforartists/*.*/datafiles && mv locale /app/share/locale && ln -sf /app/share/locale locale",
+                "install -Dm644 $FLATPAK_ID.appdata.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/Bforartists/Bforartists/releases/download/v3.2.1/Bforartists-3.2.1-Linux.tar.xz",
```

Comment:
so you unpack a binary without making the build exclusive to that architecture.

Reply:
It looks like Blender's Flathub project recently removed their `flathub.json` for some reason. https://github.com/flathub/org.blender.Blender/commit/5656ddff23b2b322698b36589f1bf448c6d7b363

I have added that file back for this submission.